core/templates/pages/teach-page/teach-page.component.html

```diff
@@ -1,137 +1,853 @@
+      <div class="oppia-teach-button-container">
+        <form action="/community-library">
+          <button type="submit" (click)="onClickBrowseLibraryButton()"
+                  class="btn oppia-teach-button oppia-teach-button-browse oppia-teach-browse-button-position oppia-transition-200"
+                  [innerHTML]="'I18N_TEACH_PAGE_LIBRARY_BUTTON' | translate">
+          </button>
+        </form>
+      </div>
+    </div>
+    <div class="oppia-teach-section-four-right" *ngIf="!isWindowNarrow">
+      <picture class="teach-with-oppia-image">
+        <source type="image/webp" [srcset]="getStaticImageUrl('/teach/oppia1.webp')">
+        <source type="image/png" [srcset]="getStaticImageUrl('/teach/oppia1.png')">
+        <img [src]="getStaticImageUrl('/teach/oppia1.png')" alt="">
+      </picture>
+    </div>
+    <div class="oppia-teach-section-four-right" *ngIf="isWindowNarrow">
+      <picture class="teach-with-oppia-image">
+        <source type="image/webp" [srcset]="getStaticImageUrl('/teach/oppia1.webp')">
+        <source type="image/png" [srcset]="getStaticImageUrl('/teach/oppia1.png')">
+        <img [src]="getStaticImageUrl('/teach/oppia1.png')" alt="">
+      </picture>
+    </div>
+    <div *ngIf="isWindowNarrow" class="oppia-teach-section-four-left mobile">
+      <h2 class="mobile-h2" [innerHTML]="'I18N_TEACH_PAGE_LIBRARY_TITLE' | translate"></h2>
+      <p class="mobile-h2" [innerHTML]="'I18N_TEACH_PAGE_LIBRARY_CONTENT' | translate"></p>
+      <div class="oppia-teach-button-container">
+        <form action="/community-library">
+          <button type="submit" (click)="onClickBrowseLibraryButton()"
+                  class="btn oppia-teach-button oppia-teach-button-browse oppia-teach-browse-button-position oppia-transition-200"
+                  [innerHTML]="'I18N_TEACH_PAGE_LIBRARY_BUTTON' | translate">
+          </button>
+        </form>
+      </div>
+    </div>
+  </div>
+
+  <div class="oppia-teach-section-five" [ngStyle]="{'background-image': 'getTestimonialBackgroundUrl()'}">
+    <div *ngIf="!isWindowNarrow" class="oppia-teach-section-five-left">
+      <h2 [innerHTML]="testimonials[displayedTestimonialId].quote | translate"></h2>
+      <h3 [innerHTML]="testimonials[displayedTestimonialId].studentDetails | translate"></h3>
+      <div class="teach-arrows-container">
+        <span (click)="decrementDisplayedTestimonialId()" class="material-icons">
+          arrow_back_ios
+        </span>
+        <span (click)="incrementDisplayedTestimonialId()" class="material-icons">
+          arrow_forward_ios
+        </span>
+      </div>
+    </div>
+    <div  *ngIf="!isWindowNarrow" class="oppia-teach-section-five-right">
+      <picture class="teach-student-image">
+        <source type="image/webp" [srcset]="testimonials[displayedTestimonialId].imageUrlWebp">
+        <source type="image/png" [srcset]="testimonials[displayedTestimonialId].imageUrl">
```

Comment:
Not sure why is it failing here, maybe you need to have the `getStaticImageUrl` in here too?

Reply:
yes, this fixed it ,Thank you.